addons/mod_tool/scripts/file_system_link.gd

```diff
@@ -0,0 +1,39 @@
+class_name FileSystemLink
+extends Reference
+
+
+# https://github.com/godot-extended-libraries/godot-next/blob/master/addons/godot-next/global/file_system_link.gd
+
+
+
+static func mk_soft_dir(p_target: String, p_linkpath: String = "") -> int:
+	var params := PoolStringArray()
+	var dir := Directory.new()
+	var output := []
+	var target := ProjectSettings.globalize_path(p_target)
+	var linkpath := ProjectSettings.globalize_path(p_linkpath)
+
+	if not dir.dir_exists(target):
+		return ERR_FILE_NOT_FOUND
+
+	match OS.get_name():
+		"Windows":
+			params.append("-command New-Item -Path")
```

Comment:
params isn't really dynamic anymore, so we might as well use this instead of many appends
```gdscript
params = [
...
]
```

Reply:
done [98273bc](https://github.com/GodotModding/godot-mod-tool/pull/106/commits/98273bc9196a8cccf2444bc38bb43c625496acf0)